blamo/lib/LogInfo/loginfo.dart

```diff
@@ -215,68 +199,98 @@ class _LogInfoPageState extends State<LogInfoPage> {
                                 focusNode: formNodes[3],
                                 attribute: 'highway',
                                 validators: [],
-                                decoration: InputDecoration(labelText: "Highway"),
+                                maxLength: 100,
+                                maxLengthEnforced: true,
+                                decoration: InputDecoration(labelText: "Highway", counterText:""),
                                 initialValue: formatValue(logInfoToBuildFrom.highway),
                                 onChanged: (void nbd){updateLogObject();},
                                 onFieldSubmitted: (v){
                                   FocusScope.of(context).requestFocus(formNodes[4]);
                                 },
                               ),
                               FormBuilderTextField(
-                                textInputAction: TextInputAction.next,
+                                textInputAction: TextInputAction.done,
                                 focusNode: formNodes[4],
                                 attribute: 'county',
                                 validators: [],
-                                decoration: InputDecoration(labelText: "County"),
+                                maxLength: 100,
+                                maxLengthEnforced: true,
+                                decoration: InputDecoration(labelText: "County", counterText:""),
                                 initialValue: formatValue(logInfoToBuildFrom.county),
                                 onChanged: (void nbd){updateLogObject();},
-                                onFieldSubmitted: (v){
-                                  FocusScope.of(context).requestFocus(formNodes[5]);
-                                },
+                              ),
+                              FormBuilderDropdown(
```

Comment:
Projections breaks, returns null value somewhere along the way resulting in: 
![image](https://user-images.githubusercontent.com/43582537/80268141-e75c0400-8659-11ea-8305-e1be580177e7.png)